Makefile

```diff
@@ -42,6 +42,16 @@ endif
 build:
 	$(GO) build $(GO_LDFLAGS) $(BUILDV) $(BUILDTAGS) -o $(PKGNAME) $(BUILDFILES)
 
+build-contained:
+	if [ -f mender-artifact ]; \
+	then \
+		echo "mender-artifact binary already exists, please move or rename"; \
+		exit 1; \
+	fi; \
+	image_id=$$(docker build -f Dockerfile . | awk '/Successfully built/{print $$NF;}'); \
```

Comment:
For getting image_id, isn't in easier to `--tag` the build to a known name?

Reply:
that might overwrite an existing tag and I wanted to avoid that... - will still remove an existing image with the same hash